lang/ast/structs.go

```diff
@@ -322,6 +360,11 @@ func (obj *StmtBind) Output(map[interfaces.Func]types.Value) (*interfaces.Output
 	return interfaces.EmptyOutput(), nil
 }
 
+// Locate stores the location from parsing in the AST node
+func (obj *StmtBind) Locate(l int, c int, el int, ec int) {
```

Comment:
Why do you need to add this method if it's an embedded struct?

Reply:
See, I thought I don't need to, but just calling `Locate` on the main structs gave me compile time errors. I don't recall the specifics. Maybe I just goofed it in some way that was not obvious to me, at the time.

Reply:
Here's a contrived example:

https://go.dev/play/p/Shwo9zOaMF8

Reply:
Fixed in 913c100